bluebrain/repo-bluebrain/packages/touchdetector/package.py

```diff
@@ -52,6 +57,10 @@ class Touchdetector(CMakePackage):
     depends_on('nlohmann-json', when='@5.3.3:')
     depends_on('intel-oneapi-tbb', when='@develop')
     depends_on('caliper@master+mpi', when='+caliper@develop')
+    depends_on('benchmark', when='+benchmark@develop')
+
+    depends_on('mvapich2', when='+asan@develop')
+    depends_on('mvapich2', when='+ubsan@develop')
```

Comment:
_I know I am nitpicking here_ but considering we typically don't add dependencies from virtual packages (`mpi`), this could be just a conflict with `hpe-mpi` if other MPI libraries like mpich/openmpi/intel-mpi could be used?

By the way, that reminds slack comment: a reproducer and relevant logs could be sent to the HPE support portal so that we can get advice/solution from the HPE team.